ui/app/mainui/ToastsManager.qml

```diff
@@ -216,7 +221,24 @@ QtObject {
             console.warn("Unexpected transaction hash while trying to navigate to the details page: " + txHash)
             return
         case ToastsManager.ActionType.OpenFirstCommunityTokenPopup:
-            console.warn("TODO: #12366")
+            if(actionData) {
+                var data = JSON.parse(actionData)
+                var communityId = data.communityId
+                var communityName = data.communityName
+                var tokenType = data.tokenType
+                var tokenName = data.tokenName
+                var tokenSymbol = data.tokenSymbol
+                var tokenImage = data.tokenImage
+                var tokenAmount = data.tokenAmount
+                Global.openFirstTokenReceivedPopup(communityId,
+                                                   communityName,
+                                                   rootChatStore.getCommunityDetailsAsJson(communityId).image,
```

Comment:
Can we receive the community image from the backend as well?

Reply:
@Cuteivist do you plan to expose it from nim, or should we create a separate task for this?

Reply:
@Cuteivist if this pr is merged before you finish with the backend, feel free to update the UI accordingly!